com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/cf/v2/ApplicationEnvironmentCloudModelBuilder.java

```diff
@@ -24,11 +24,13 @@
 
     protected final DeploymentDescriptor deploymentDescriptor;
     protected final String deployId;
+    protected final String namespace;
     protected final boolean prettyPrinting;
 
-    public ApplicationEnvironmentCloudModelBuilder(DeploymentDescriptor deploymentDescriptor, String deployId, boolean prettyPrinting) {
+    public ApplicationEnvironmentCloudModelBuilder(DeploymentDescriptor deploymentDescriptor, String deployId, String namespace, boolean prettyPrinting) {

```

Comment:
Do you think that it will be better if we use builder for this class?

Reply:
I agree that some sort of refactoring could be useful here, but also didn't want to extend the scope once again